modules/dcache/src/main/java/org/dcache/services/ssh2/Ssh2Admin.java

```diff
@@ -201,11 +208,42 @@ private void addOrigin(ServerSession session, Subject subject)
         }
     }
 
+    private void logLoginTry(String username, ServerSession session,
+                             String method, boolean successful, String reason)
+    {
+        NetLoggerBuilder.Level logLevel = NetLoggerBuilder.Level.INFO;
+        if(!successful) {
```

Comment:
Since you are altering a pre-existing file, you probably should try to conform to the style of the file.  Here there probably should be a space between if and (.